starport/pkg/protopath/protopath.go

```diff
@@ -2,22 +2,22 @@ package protopath
 
 import (
 	"errors"
-	"os"
 	"path/filepath"
 
+	"github.com/tendermint/starport/starport/pkg/xfilepath"
+
```

Comment:
```suggestion
```

+ fmt.

Reply:
I actually let Golanfd managing the import and unfortunately constantly add empty lines

Reply:
Yeah, it happens. Empty lines in imports section is actually for grouping different kind of dependencies. But we don't do that so these ones are just dangling I think.